index.html

```diff
@@ -10,17 +11,28 @@
     </head>
     <body>
         <h1>Paths to Citizenship</h1>
+        <div id="map" class="viz">
+            <h2>Migration is natural</h2>
+            <p>Eventually we will replace the dropdown menu with a clickable map!</p>
             <div class="container" style="display:inline-block">
```

Comment:
The dropdown is also temporary. I kept it in so that we can work on the interactivity across the whole page while the map is in development.